README.md

```diff
@@ -75,15 +75,15 @@ Load the template in your preamble:
 ```
 
 ### Package options
-
+```
 | Option      | Description                                                       | Default |
```

Comment:
Ouch, sorry. Different Markdown implementations use different characters for tables. I previewed the README with Pandoc but forgot to enable the GitHub markdown mode.

To fix the table for display on GitHub, change the `+` symbols in the header separator to `|`.